sdk/core/core-rest-pipeline/src/policies/throttlingRetryPolicy.ts

```diff
@@ -13,7 +13,17 @@ export const throttlingRetryPolicyName = "throttlingRetryPolicy";
 /**
  * Maximum number of retries for the throttling retry policy
  */
-export const DEFAULT_CLIENT_MAX_RETRY_COUNT = 3;
+export const DEFAULT_CLIENT_MAX_RETRY_COUNT = 10;
```

Comment:
@xirzec I updated this to 10 to be consistent with the other policies. I also found Python to be doing the same. What do you think?

Reply:
Hmm.. I mean technically it is a behavioral change, but I guess I'm not too worried about it, as long as it is configurable